deployments/deploy.go

```diff
@@ -52,8 +57,12 @@ func Deploy(app, archivePath, gitRef string) error {
 
 	scalingoio.Status("Your deployment has been queued and is going to start…")
 
-	go showQueuedWarnings(c, app, deployment.ID)
+	if opts.NoFollow {
+		scalingoio.Status(fmt.Sprintf("The no-follow flag has been passed. you can check deployment logs with scalingo -a %s deployment-follow", app))
```

Comment:
nitpick: 
- replace the use of `Status(fmt.Sprintf(` with `Statusf`
- in informative messages targeted to users, I prefer the use of long options. This is for example the case in our documentation: https://doc.scalingo.com/platform/app/logs
- (i'm not sure about this one.. 😅 ) I think "is passed" is a better conjugation than "has been passed".
- Missing upper case

```suggestion
		scalingoio.Statusf("The no-follow flag is passed. You can check deployment logs with scalingo --app %s deployment-follow", app)
```